src/udp_socket.rs

```diff
@@ -84,6 +84,9 @@ impl UdpSocket {
     pub async fn send_to(&self, buf: &[u8], addr: SocketAddr) -> Result<(), Error> {
         self.split().1.send_to(buf, addr).await
     }
+    pub async fn send(&self, buf: &[u8]) -> Result<(), Error> {
```

Comment:
This only works if the UDP socket is `connected`. Currently there's no API implemented for this. So probably best to remove this for now.

Reply:
again, a leftover from hacking around. It shouldn't have entered the pr. dtls is not using the udpsocket anyway.